Cartfile.resolved

```diff
@@ -1,19 +1,17 @@
-binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseAnalyticsBinary.json" "8.9.0"
-binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseCrashlyticsBinary.json" "8.9.0"
+binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseAnalyticsBinary.json" "8.9.1"
+binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseCrashlyticsBinary.json" "8.9.1"
+github "NYPL-Simplified/CardCreator-iOS" "d6745eed41e6e6439024a23a74236f9f03a01d59"
 github "NYPL-Simplified/NYPLAudiobookToolkit" "3d6b92b4294259cfe393660a2e1de206264fcb31"
 github "NYPL-Simplified/PDFRendererProvider-iOS" "d63e6ba168cb91617a4aca5e306a92af1dd4497c"
 github "NYPL-Simplified/audiobook-ios-overdrive" "6e882e29a62d3a4e2cebe04a8943ed3855bcb599"
-github "NYPL-Simplified/CardCreator-iOS" "d6745eed41e6e6439024a23a74236f9f03a01d59"
 github "PureLayout/PureLayout" "v3.1.9"
 github "cezheng/Fuzi" "3.1.3"
 github "dexman/Minizip" "1.4.0"
-github "krzyzanowskim/CryptoSwift" "1.4.1"
+github "krzyzanowskim/CryptoSwift" "1.4.2"
+github "ra1028/DifferenceKit" "1.2.0"
```

Comment:
Why do we need this framework? Just wondering

Reply:
this was added by the new R2. It seems like it's required by the Navigator:
https://github.com/readium/swift-toolkit#carthage